website/source/docs/templates/engine.html.md

```diff
@@ -72,7 +72,7 @@ Here is a full list of the available functions for reference.
           "inline": ["echo $TESTVAR"]
         },
     ```
-    Valid variables to request are: "ID", "Host",
+    Valid variables to request are: "ID", "SourceImageName", "Host",
```

Comment:
I think it could be nice to point that this will correspond to source ami name in amazon and the source image of the builder for other builders ? May be adding one a few more words saying this currently only works for amazon ?

Reply:
I think this is the wrong place to put this in the docs; moving forward, some of these values are going to be builder-specific so we should have a section in the builder docs for custom template variables.  

Then, here, we can say "please also refer to the builder docs for builder-specific engine variables"

Reply:
Oh yes ! I agree 🙂